javaparser-core/src/main/java/com/github/javaparser/ast/validator/language_level_validations/Java1_0Validator.java

```diff
@@ -138,4 +139,19 @@ public Java1_0Validator() {
         add(noSealedClasses);
         add(noPermitsListInClasses);
     }
+
+    /**
+     * Compose a validator message with suggestion to enable a feature.
+     * @param cause Original message - cause of the error.
+     * @param level Language level to enable.
+     * @return Composed message.
+     */
+    private static String message(final String cause, final ParserConfiguration.LanguageLevel level){
+        return String.format(
+            "%s To enable this feature, add this line before parsing the Java file: StaticJavaParser.getParserConfiguration().setLanguageLevel(%s).",
```

Comment:
Hi, Thank you for your rpoposal but StaticJavaParser is a utility class that cannot be pushed forward. It should simply be specified that the language level must be configured in the configuration before parsing the source files.

Reply:
@jlerbsc I changed the message description. Thank you.